hackathon_site/event/views.py

```diff
@@ -203,3 +205,13 @@ class TeamListView(mixins.ListModelMixin, generics.GenericAPIView):
 
     def get(self, request, *args, **kwargs):
         return self.list(request, *args, **kwargs)
+
+
+class TeamIncidentDetailView(mixins.RetrieveModelMixin, generics.GenericAPIView):
+    queryset = Incident.objects.all()
+
+    serializer_class = IncidentListSerializer
+    permission_classes = [FullDjangoModelPermissions]
+
+    def get(self, request, *args, **kwargs):
+        return self.retrieve(request, *args, **kwargs)
```

Comment:
You need to check whether the incident being queried belongs to the user's team, if it doesn't, then throw an unauthorized error.

Also, update the unit tests for both these comments.

Reply:
- [ ] Following up